Source/abci/Importer/aiContext.cpp

```diff
@@ -39,7 +39,28 @@ static std::string NormalizePath(const char *in_path)
     return S(path);
 }
 
+#ifdef WIN32
 
+class lockFreeIStream : public std::ifstream
+{
+private:
+	FILE *_file = nullptr;
+public: 
+	lockFreeIStream(const wchar_t  *name) :_file( _wfsopen(name, L"rb", _SH_DENYNO)), 
+                                               std::ifstream(_file)
```

Comment:
You are opening the file with _SH_DENYNO, Which means as we read the file, someone can also read and write to it. What would happen if someone write to this file while you are reading from it?

Reply:
the ABC importer is detecting file delete/moves and closes all files in this case.
Need to test if I can replace while actually playing.